src/dune_rules/dune_file.ml

```diff
@@ -1761,8 +1748,8 @@ module Rule = struct
       | Action -> short_form
       | Since (version, inner) ->
         let what = Printf.sprintf "'%s' in short-form 'rule'" atom in
-        let* () = Dune_lang.Syntax.since ~what Stanza.syntax version in
-        interpret atom inner
+        Dune_lang.Syntax.since ~what Stanza.syntax version
+        >>> interpret atom inner
```

Comment:
why did you make this change?

Reply:
Is it not the same thing? The idea was to make it more similar to the rest of the code.

Reply:
But if you are asking "why make this change here", it is part of the second commit. If you like I can split that into another PR.

Reply:
Yes to both points.

First, I don't see how this change is an improvement. Second, it's hard to review PR's that mix unrelated changes.

Reply:
OK I've left only the first commit.